pandas/core/dtypes/dtypes.py

```diff
@@ -122,21 +120,12 @@ def __unicode__(self):
     def __str__(self):
         """
         Return a string representation for a particular Object
-
```

Comment:
we might not even need to define str and unicode here and in base.py (future PR)